projects/portal/src/app/pipes/decimals.pipe.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { DecimalsPipe } from './decimals.pipe';
+
+describe('DecimalsPipe', () => {
+  const pipe = new DecimalsPipe();
+
+  it('Extract 3 decimal place', () => {
+    expect(pipe.transform('123.456', 3)).toBe('.456');
+  });
+
+  it('Set a number with more than a decimal point', () => {
+    expect(pipe.transform('123.456', 4)).toBe('.456');
+  });
+
+  it('When input a string', () => {
```

Comment:
The name of the test should be one sentence with it, especially when written in English.

For example, `return empty string when input a string` rather than `When input a string`.
This way, when `it` is also included, it becomes a sentence `it return empty string when input a string`.

Reply:
Thank you, I fix the comment.